app.py

```diff
@@ -107,5 +115,4 @@ def refresh():
 
 
 with app.app_context():
-    refresh()
-    refresh()
+    refresh(app.config['REFRESH_PASSWORD'])
```

Comment:
We should call `refresh` two times here to solve the empty Redis cache issue in the first startup of the application.

Reply:
Yes @zaidalyafeai , should I handle the comments that @AliOsm mentioned above, Inshallah will end from that by tomorrow.